plugins/sigma.renderers.customShapes/sigma.renderers.customShapes.canvas.js

```diff
@@ -32,10 +27,7 @@
         image = document.createElement('IMG');
```

Comment:
Let's create a setting called `imgCrossOrigin` with default `anonymous` for developers who want more security.